src/components/ConfirmationDialog/ConfirmationDialog.tsx

```diff
@@ -1,30 +1,59 @@
-import classNames from "classnames";
-import {Button} from "components/Button";
-import {ScrumlrLogo} from "components/ScrumlrLogo";
+import React from "react";
+import {ReactComponent as CloseIcon} from "assets/icon-close.svg";
+import {Portal} from "components/Portal";
+import {animated, Transition} from "react-spring";
+import {useTranslation} from "react-i18next";
 import "./ConfirmationDialog.scss";
 
-export interface ConfirmationDialogProps {
-  headline: string;
-  acceptMessage: string;
-  declineMessage: string;
+type ConfirmationDialogProps = {
+  title: string;
   onAccept: () => void;
+  onAcceptLabel?: string;
   onDecline: () => void;
-  className?: string;
-}
+  onDeclineLabel?: string;
+};
 
-export const ConfirmationDialog = ({headline, acceptMessage, onAccept, declineMessage, onDecline, className}: ConfirmationDialogProps) => (
-  <div className={classNames("confirmation-dialog__wrapper", className)}>
-    <div className="confirmation-dialog">
-      <ScrumlrLogo accentColorClassNames={["accent-color--blue", "accent-color--purple", "accent-color--lilac", "accent-color--pink"]} />
-      <h2>{headline}</h2>
-      <Button type="submit" color="primary" className="confirmation-dialog__button" onClick={() => onAccept()}>
-        {acceptMessage}
-      </Button>
-      <Button type="submit" color="primary" className="confirmation-dialog__button" variant="outlined" onClick={() => onDecline()} autoFocus>
-        {declineMessage}
-      </Button>
-    </div>
-  </div>
-);
+export const ConfirmationDialog: React.FC<ConfirmationDialogProps> = (props) => {
+  const {t} = useTranslation();
+  const transitionConfig = {
```

Comment:
might make sense to unify this config with the one for the settings dialog, as I think they should always be identical